src/Ecosystem/templates/list.phtml

```diff
@@ -0,0 +1,187 @@
+                                <?php else : ?>
+                                    <li><a class="package-button type dropdown-item" data-value="<?= $case->value ?>" href="#"><?= $case->name ?></a></li>
+                                <?php endif; ?>
+                            <?php endforeach ?>
+                        </ul>
+                    </div>
+
+                    <div class="btn-group">
+                        <button type="button" class="btn btn-outline-success">Usage</button>
+                        <button type="button" class="btn btn-outline-success dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown" aria-expanded="false">
+                            <span class=""><?= $usage ?? '' ?></span>
+                        </button>
+                        <ul class="dropdown-menu">
+                            <?php foreach (EcosystemUsageEnum::cases() as $case): ?>
+                                <?php if ($case->name === $usage): ?>
+                                    <li><a class="package-button usage dropdown-item active" data-value="<?= $case->value ?>" href="#"><?= $case->name ?><i class="bi bi-x-circle"></i></a></li>
+                                <?php else : ?>
+                                    <li><a class="package-button usage dropdown-item" data-value="<?= $case->value ?>" href="#"><?= $case->name ?></a></li>
+                                <?php endif; ?>
+                            <?php endforeach ?>
+                        </ul>
+                    </div>
+
+                    <div class="btn-group">
+                        <button type="button" class="btn btn-outline-success">Category</button>
+                        <button type="button" class="btn btn-outline-success dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown" aria-expanded="false">
+                            <span class=""><?= $category ?? '' ?></span>
+                        </button>
+                        <ul class="dropdown-menu">
+                            <?php foreach (EcosystemCategoryEnum::cases() as $case): ?>
+                                <?php if ($case->name === $category): ?>
+                                    <li><a class="package-button category dropdown-item active" data-value="<?= $case->value ?>" href="#"><?= $case->name ?><i class="bi bi-x-circle"></i></a></li>
+                                <?php else : ?>
+                                    <li><a class="package-button category dropdown-item" data-value="<?= $case->value ?>" href="#"><?= $case->name ?></a></li>
+                                <?php endif; ?>
+                            <?php endforeach ?>
+                        </ul>
+                    </div>
+
+                    <div class="btn-group">
+                        <button id="clear-filters-button" type="button" class="btn btn-outline-danger">Clear filters <i class="bi bi-x-circle"></i></button>
```

Comment:
The button to clear the filter should only be visible if the user has set a filter and should also reset the search. Otherwise I find the handling confusing.